tools/harness-thci/OpenThread.py

```diff
@@ -3489,6 +3493,7 @@ def registerMulticast(self, sAddr='ff04::1234:777a:1', timeout=MLR_TIMEOUT_MIN):
         Args:
             sAddr   : str : Multicast address to be subscribed and notified OTA.
         """
+        self._beforeRegisterMulticast(sAddr, timeout)
```

Comment:
This method _beforeRegisterMulticast is defined on OpenThread_BR it's not available in OpenThread how have you managed to call it? This would work on BR but not on normal embedded device.

Reply:
That method is added as part of this PR.  See line #325.

Reply:
I misunderstood your question the first time.  You are correct, I missed that one line of the merge.  I'll add that in.

Reply:
Looks ok.